client/structs/structs.go

```diff
@@ -184,3 +187,87 @@ func (d *DriverNetwork) Hash() []byte {
 	}
 	return h.Sum(nil)
 }
+
+type FingerprintRequest struct {
+	Config *config.Config
+	Node   *structs.Node
+}
+
+type FingerprintResponse struct {
+	attributes map[string]string
+	links      map[string]string
+	resources  *structs.Resources
```

Comment:
Should we have a `Applicable bool` in the response so we can maintain our old behavior of displaying whether the fingerprinter is being applied.

Reply:
Sure, we could do that, that would probably be cleaner than parsing attribute values after the fact.

Reply:
On second thought- would logging drivers where `fingerprintSuccess` is true achieve the same goal?